packages/web-components/src/components/masthead/masthead-logo.ts

```diff
@@ -35,15 +45,66 @@ class DDSMastheadLogo extends FocusMixin(LitElement) {
   @property({ reflect: true })
   slot = 'brand';
 
+  /**
+   * Search bar opened flag.
+   */
+  @internalProperty()
+  private _openedSearch = false;
+
+  /**
+   * Large breakpoint flag.
+   */
+  @internalProperty()
+  private _responsiveLg = false;
+
+  /**
+   * Handles toggle event from the search component.
+   *
+   * @param event The event.
+   */
+  @HostListener(`parentRoot:${DDSMastheadSearch.eventToggle}`)
```

Comment:
To support alternate (typically) component composed in masthead, this codebase makes event name configurable in receiving component side ([an example](https://github.com/carbon-design-system/carbon-web-components/blob/v1.5.0/src/components/ui-shell/side-nav.ts#L198-L200)). So please define `eventToggle` static property to each classes receiving the event:

```suggestion
  @HostListener(`parentRoot:eventToggle`)
```